### ISSUE TITLE
[Enhancement] Function to clear value from store

### ISSUE THREAD

**Issue by rishabhsaxena** [appsmithorg/appsmith#7827]
## Description
By default, storeValue automatically save its values to localStorage . Passing in false afterwards won't delete the old value, you have to open devtools and clear it from localStorage

**Comment by Nikhil-Nandagopal** [comment]
I don't think false should clear the store but we should have an action to clear the store

**Comment by marks0351** [comment]
another user on discord has asked for this action. it also seems useful coz i wished we had this when i was building on of my applications. we can also show on the entity explorer the store's tree coz its very difficult to see what the store value is at the very moment.
cc: @Nikhil-Nandagopal @rishabhsaxena

**Comment by Nikhil-Nandagopal** [comment]
☝️ @ajinkyakulkarni @hetunandu @ApekshaBhosale

**Comment by pranavkanade** [comment]
![Screenshot 2022-01-31 at 4 04 31 PM](https://user-images.githubusercontent.com/13262095/151778563-0b8ccfb4-927a-40c8-83af-97507ef04183.png)
+1

**Comment by Nikhil-Nandagopal** [comment]
@ajinkyakulkarni we've had a couple of these requests. Can we introduce a clearValue function?

**Comment by marks0351** [comment]
posting a js object work around for now, but definitely need a built in action.

```
 clearStorage: () => {
        Object.keys(appsmith.store).forEach((eachKey)=>{
            storeValue(eachKey, undefined)
        })
    }
```

**Comment by rohan-arthur** [comment]
this will be fixed with #11639
